debug/init.js

```diff
@@ -21,6 +21,7 @@ $(function(){
       .selector('node')
         .css({
           'label': 'data(id)',
+          'shape': 'barrel',
```

Comment:
Debug page changes shouldn't be committed unless it's adding a debug feature etc.